### ISSUE TITLE
Hot code push in Eliom.

### ISSUE THREAD

**Issue by dannywillems** [ocsigen/ocsigen-start#261]
To update the mobile application, the Cordova plugin [cordova hot code push](https://github.com/nordnet/cordova-hot-code-push) (CHCP) is used.
It works well on Android and iOS but not on other platforms like Windows Phone or Blackberry. It implies Ocsigen-start can't be build for these platforms with Cordova.

As other platforms are not planned, it causes some problems to be compatible with all platforms.

A solution is to write an equivalent library (with Eliom), maybe independtly of Eliom to update automatically the application with the same (or improved) process than CHCP.

**Comment by vasilisp** [comment]
That would be a lot of work, including entirely custom code for platforms of marginal importance.

**Comment by dannywillems** [comment]
@vasilisp Yes, we talk a few minutes about it with @vouillon @balat and @pwbs. I prefer to add the discussions result as an issue with the label `enhancement` to have it somewhere.
I think it is also a good way to help people to know how they can contribute if they want.

**Comment by sagotch** [comment]
Note: I don't like CHCP at all, and would prefer to have a real upgrade from the store for my app. Please include CHCP only if it is really easy to be removed...

**Comment by sagotch** [comment]
Starting a new project with `ocsigen-start` (actually trying to upgrade old code using EBA), I find it to difficult to get rid of everything I do not want, in particular, I have no idea about how to remove this awful chcp thing...

**Comment by dannywillems** [comment]
@sagotch Do you need mobile application?

**Comment by sagotch** [comment]
Yes

**Comment by sagotch** [comment]
See https://github.com/ocsigen/ocsigen-start/issues/419
